specification/maps/data-plane/Render/preview/2.1/render.json

```diff
@@ -213,18 +196,305 @@
             "type": "array",
             "collectionFormat": "csv",
             "items": {
-              "type": "string"
+              "type": "number",
+              "format": "double"
+            }
+          }
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/MapAttribution"
             }
+          },
+          "default": {
+            "$ref": "../../../Common/preview/1.0/common.json#/responses/default"
+          }
+        }
+      }
+    },
```

Comment:
Is this endpoint missing: `/map/static/{format}`?

Reply:
correct, `/map/imagery/` has been incorrect added instead of /map/static, thank you